governance/delegates/0x4D8C9c2Cd8846533967b07BECFC437542196Ad6A/profile.md

```diff
@@ -1,25 +1,50 @@
+
+These are the objectives, and only these.
+
+This does not mean that, for example, MakerDAO should not be active in some form of charity. To the contrary this could provide positive rewards to MKR-holders as hinted above. But each decision/activity/endeavour **must** always be able to clearly answer to the question: "*How does it contribute to objectives 1 and 2?*". A scientific approach should always be used to determine the likelihood of success of any given proposal.
+
+**How to achieve the goals?**
+1) Embrace complexity.
+"There is always a simple solution to any problem, and that is often the wrong one." 
+This means that MakerDAO is a VERY complex project, and cannot be dumbed down to a simple receipt for success. This is clearly witnessed by the fact that despite all, all other "decentralised stablecoins" emerged so far, are either orders of magnitude smaller that DAI or are basically just scams.
+2) Pretend simplicity.
+Also the construction and deployment of a stellar probe, like the Voyager program, is an extremely complex endeavour. This however does not mean that it cannot be explained to people, at different levels of understanding and details, in a way that positively impact everybody. In particular, the Voyager program has been _successfully managed and directed_ (also) by people without a PhD in physics.
+3) Keep it simple stupid! (KISS)
+Simple yet effective structures can be used to understand and manage extremely complex projects. Keeping structures simple and transparent and avoiding the (often natural) "complexity explosion" is not easy. It needs constant attention and work.
+
+The KISS AVC will use its influence and power to achieve Goals 1&2 by means of the KISS approach: embrace complexity, pretend simplicity, and maintain simple structures. 
+
+The two Goals 1&2 are clear. Everything that has a scientifically proven likelihood of achieving these goals, within the given amount of resources, will be supported. All the rest will be rejected as "out of focus for now".
+
+**Where do we sit in the Dove/Hawk spectrum?**
+
+The ENDGAME plan proposed by Rune is compatible with the KISS principle when it requires AVCs to be broadly classified in two categories: Doves or Hawks.
+
+Of course, a binary description of this type does not capture the immense variety of viewpoints of all AVCs, not to mention of the MKR-holders within a given AVC. But it is useful in simplifying the landscape (KISS!)
+
+**=>** The KISS AVC sits on the **Hawkish** side of the spectrum.
 
 # Penguin Soldier
-Delegate Address: 0x07621bd61baeffc3cbcbec511cb1cbaf3c5dfafa
+Delegate Address: 0x4d8c9c2cd8846533967b07becfc437542196ad6a
```

Comment:
@blimpa this address is incorrect and needs to be reverted to original address. 

The Delegate Address field here is NOT asking for the Delegation Contract address. 

It's asking for the AD's wallet address that controls the Delegation Contract. You can see this by going to the Forum [post](http://forum.makerdao.com/t/penguin-soldier-ad-recognition-submission/21395), and clicking on the Etherscan [link](https://etherscan.io/verifySig/25915) for KISS AVC delegation contract. 

Look at the top field under **Address.** That's the contract address that should go into this field.